lib/matplotlib/tests/test_category.py

```diff
@@ -187,6 +187,65 @@ def test_plot_1d_missing(self):
         self.axis_test(ax.yaxis, self.dmticks, self.dmlabels, self.dmunit_data)
 
     @cleanup
+    @pytest.mark.usefixtures("data")
+    def test_plot_bytes(self):
+        counts = np.array([4, 6, 5, 1])
+        fig, ax = plt.subplots(ncols=3)
+
+        ax[0].bar(self.d, counts)
+
+        types = [v.encode('ascii') for v in self.d]
```

Comment:
hmm, maybe this should just be explicit then? `[b'a', b'c', b'c', b'd']` I think I'm advocating against using the fixture since it's not really used here.